src/libraries/System.Memory/ref/System.Memory.cs

```diff
@@ -345,12 +349,16 @@ public static partial class BinaryPrimitives
         public static uint ReverseEndianness(uint value) { throw null; }
         [System.CLSCompliantAttribute(false)]
         public static ulong ReverseEndianness(ulong value) { throw null; }
```

Comment:
Missing overloads `ReverseEndianness(double)` and `ReverseEndianness(float)` in the ref assemblies.